parity/secretstore.rs

```diff
@@ -64,10 +66,35 @@ mod server {
 	impl KeyServer {
 		/// Create new key server
 		pub fn new(conf: Configuration, _deps: Dependencies) -> Result<Self, String> {
+			let key_pairs = vec![
+				ethkey::KeyPair::from_secret(ethkey::Secret::from_str("6c26a76e9b31048d170873a791401c7e799a11f0cefc0171cc31a49800967509").unwrap()).unwrap(),
+				ethkey::KeyPair::from_secret(ethkey::Secret::from_str("7e94018b3731afdb3b4e6f4c3e179475640166da12e1d1b0c7d80729b1a5b452").unwrap()).unwrap(),
+				ethkey::KeyPair::from_secret(ethkey::Secret::from_str("5ab6ed2a52c33142380032c39a03a86b12eacb3fa4b53bc16d84f51318156f8c").unwrap()).unwrap(),
+			];
 			let conf = ethcore_secretstore::ServiceConfiguration {
-				listener_addr: conf.interface,
-				listener_port: conf.port,
-				data_path: conf.data_path,
+				listener_address: ethcore_secretstore::NodeAddress {
+					address: conf.interface.clone(),
+					port: conf.port,
+				},
+				data_path: conf.data_path.clone(),
+				// TODO: this is test configuration. how it will be configured in production?
+				cluster_config: ethcore_secretstore::ClusterConfiguration {
+					threads: 4,
+					self_private: (***key_pairs[(conf.port - 8082) as usize].secret()).into(),
+					listener_address: ethcore_secretstore::NodeAddress {
+						address: conf.interface.clone(),
+						port: conf.port + 10,
+					},
+					nodes: key_pairs.iter().enumerate().map(|(i, kp)| (kp.public().clone(),
+						ethcore_secretstore::NodeAddress {
+							address: conf.interface.clone(),
+							port: 8082 + 10 + (i as u16),
```

Comment:
Yes - it is decribed couple of lines above :)
```
// TODO: this is test configuration. how it will be configured in production?
```
It is question №1 in this PR. Basically - I didn't knew whether configuration should be dynamic or static. But still wanted to make some kind of demonstration in real environment (not in tests). So here's test configuration for 3 nodes, which have 2 ports with diff = 10.
Now as I know it must be static (at least for now), I have already removed this test config in my local branch, but still waiting for this PR to be closed before making next one :)